converters/2channels2panoptic_coco_format.py

```diff
@@ -64,7 +64,7 @@ def convert_single_core(proc_id, image_set, categories, source_folder, segmentat
             segment_id, color = id_generator.get_id_and_color(sem)
             pan_format[mask] = color
             segm_info.append({"id": segment_id,
-                              "category_id": sem})
+                              "category_id": sem.item()})
```

Comment:
```suggestion
                              "category_id": int(sem)})
```

Will this work? In another script in this repo I used `int()` so may be a little more consistent?

Reply:
Thanks a lot! int() also works!